docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -375,7 +399,8 @@ h|Kafka property name
 |===
 --
 
-
+////
+// Removing for now per epic brief: "message.format.version: Don't allow setting it even to default". (Stetson, 13 June 2022)
```

Comment:
Luke, I've removed this `message.format.version` property per the epic brief, but see my last comment about message down-conversion, which relates to this and thus makes me wonder why we wouldn't keep this property in support.

Reply:
@showuon So you're sure it's okay to remove this property since we've also introduced down-conversion, which is related? Per our other conversation in that comment.

Reply:
Yes, `message.format.version` property is uneditable. So does that mean we should remove it on the doc? I'm not sure.

Reply:
@sterobin , this is the only question I have, otherwise, LGTM. Thanks.

Reply:
@showuon Well, there's a difference between "uneditable" vs. what the Jira says for this property, which is "Don't allow setting it **even to default**" So not not enable users to even set the default means that we don't support this property, and therefore it should be removed. But if that wording is innaccurate, and if what we really mean is "This property is uneditable" and the default = 2.7-IV2`, then yes, we can should keep.

Is there a way you can confirm whether this property is uneditable vs. unsupported? If supported but simply uneditable, can you verify if the default value is still `2.7-IV2`?

Thanks.

Reply:
It's `uneditable`, and default value is `3.0-IV1`. The default value should not be updated because this config is deprecated now. Thanks.

Reply:
Sorry, on second thought, I think you were right. Since we don't allow them to set it, even to default value. So it's `unsupported`. And I agree we don't have to put it in doc. Thanks.

Reply:
@showuon ack. I've restored the property and updated that default. Thank you!

Reply:
Anyway, since you've added it as `uneditable`, I don't think it is wrong. After all, they are the `read-only` property. So, let's keep it. Thanks.